core/field_dropdown.js

```diff
@@ -252,6 +252,15 @@ Blockly.FieldDropdown.prototype.showEditor_ = function() {
   var menu = new goog.ui.Menu();
   menu.setRightToLeft(this.sourceBlock_.RTL);
   for (var i = 0; i < options.length; i++) {
+    var separator = options[i] == 'SEPARATOR';
```

Comment:
@rachel-fenichel I'm not a huge fan of this, do you have any recommendations here or does this seem fair to you?

Reply:
I have not, but yeah that makes sense. I'll fix that up.